### PR TITLE
[ONNX][FRONTEND] Constantfill - #1539

### PR DESCRIPTION
**Pull request by srkreddy1238**

### CODE REVIEW

**Comment by yzhliu** [comment]
Please request review.

**Comment by srkreddy1238** [comment]
cc @siju-samuel @nishi-t welcome to review.

File: nnvm/python/nnvm/frontend/onnx.py [hunk 1]
```diff
@@ -611,6 +611,46 @@ def _impl_v1(cls, inputs, attr, params):
...
+        num_inputs = len(inputs)
+        if 'shape' in attr:
+            if num_inputs > 0:
+                raise ImportError(
+                    "Can't set shape and input tensor at a time")
+            shape = attr.pop('shape')
+        else:
+            if num_inputs == 0:
+                raise ImportError(
+                    "Either shape attribute or input should be set")
```
Comment (nishi-t) [inline nnvm/python/nnvm/frontend/onnx.py]: Does this code work when `num_inputs` is greater than 1? If not, I'd suggest raising the error also when num_inputs is not 1.

**Comment by yzhliu** [comment]
Thanks @srkreddy1238 @nishi-t

**Pull request merged**